src/main/java/Duke.java

```diff
@@ -6,5 +8,33 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        greet();
+        Scanner scanner = new Scanner(System.in);     
+
+        user_active: while (true) {
+            String usercommand = scanner.nextLine();
+            String[] commands = usercommand.split(" ");
+
+            switch (commands[0]) {
+                case "bye":
+                    byebye();
+                    break user_active;
```

Comment:
According to the module's coding standard, there should not be an indentation for case clauses